Source/FlowEditor/Public/Graph/Nodes/FlowGraphNode.h

```diff
@@ -202,4 +216,107 @@ class FLOWEDITOR_API UFlowGraphNode : public UEdGraphNode
 
 	virtual EFlowSignalMode GetSignalMode() const;
 	virtual bool CanSetSignalMode(const EFlowSignalMode Mode) const;
+
+//////////////////////////////////////////////////////////////////////////
```

Comment:
SubNode support grafted on from the EnvironmentQuery graph editor (mostly) as a guide, and then modified from there to blend it in with the existing FlowEditor.